ui/lib/apps/SlowQuery/translations/en.yaml

```
@@ -34,22 +34,45 @@ slow_query:
     db: Execution Database
     db_tooltip: The database used to execute the query
 
-    parse_time: Parse Time
+    query_time2: Query Time
+    query_time2_tooltip: The execution time of a query (due to the parallel execution, it may be significantly smaller than the following time)
+    parse_time: 　　Parse Time
     parse_time_tooltip: Time consumed when parsing the query
-    compile_time: Compile Time
-    compile_time_tooltip: Time consumed when optimizing the query
-    wait_time: Coprocessor Wait Time
-    process_time: Coprocessor Process Time
-    backoff_time: Backoff Retry Time
+    compile_time: 　　Generate Plan Time
+    rewrite_time: 　　　　Rewrite Plan Time
+    preproc_subqueries_time: 　　　　　　Preprocess Sub-Query Time
+    preproc_subqueries_time_tooltip: The time consumed of pre-process the subquery during the rewrite plan phase
+    optimize_time: 　　　　Optimize Plan Time
+    wait_ts: 　　Get Txn Start Ts Time
```

```suggestion
    wait_ts: 　　Get Start Ts Time
```

(to be aligned with "Get Commit Ts Time" below)